icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -1602,8 +1602,8 @@ class SegmentationRule {
         NO_BREAK = u'×',
     };
     struct BreakContext {
-        BreakContext(std::int64_t index) : indexInRemapped(index) {}
-        std::optional<std::int64_t> indexInRemapped;
+        BreakContext(int index) : indexInRemapped(index) {}
```

Comment:
In ICU, we normally use int32_t not int. In the old days, an int could be just 16 bits wide. I suppose it could become 64 bits on some machine. Anyway, the functions involved explicitly return int32_t, so that would guarantee a type match.

Reply:
This file is full of `int`s, but done.

(But then if we are not going with the concision of `int`, why not `int64_t`? It’s not like there is a performance benefit these days, and we did [get bitten something being 32-bit here](https://github.com/unicode-org/icu/commit/cc64ec7c94a822ae97dea5cd5a4fa972b2e465ef)…)

Reply:
Where 32 bits are plenty, I still prefer to use them. It's also consistent with long-standing ICU practice, and consistent with Java whose "int" is precisely int32_t, and is used for string indexes etc., so can help a little with porting between the languages.

Reply:
... and we universally use `int32_t` not `std::int32_t` but I won't block the PR on that now...

Reply:
> ... and we universally use int32_t not std::int32_t but I won't block the PR on that now...

But I did it anyway, so now you get to restamp the rubber :-p